single-sign-on/configure/index.md

```diff
@@ -0,0 +1,158 @@
+</ul>
+<div class="tab-content">
+<div id="windows" class="tab-pane fade in active" markdown="1">
+
+#### SAML 2.0
+
+1. Sign in to [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} as an administrator and navigate to **Organizations** and select the organization that you want to enable SSO on.
+2. Select **Settings** and select the **Security** tab.
+3. Select an authentication method for **SAML 2.0**.
+
+    ![SSO SAML1](/single-sign-on/images/sso-saml1.png){:width="500px"}
+
+4. In the Identity Provider Set Up, copy the **Entity ID**, **ACS URL** and **Certificate Download URL**.
+
+    ![SSO SAML2](/single-sign-on/images/sso-saml2.png){:width="500px"}
+
+5. Sign in to your IdP to complete the IdP server configuration process. Refer to your IdP documentation for detailed instructions.
+
+    > **Note**
+    >
+    > The NameID is your email address and is set as the default.
+    > For example, yourname@mycompany.com. The optional `name` attribute is also supported. This attribute name must be lower-cased. _The following is an example of this attribute in Okta._
+
+    ![SSO Attribute](/single-sign-on/images/sso-attribute.png){:width="500px"}
+
+6. Complete the fields in the **Configuration Settings** section and select **Save**. If you want to change your IdP, you must delete your existing provider and configure SSO with your new IdP.
+
+    ![SSO SAML3](/single-sign-on/images/sso-saml3.png){:width="500px"}
+
+7. Proceed to **add your domain** before you test and enforce SSO.
+
+<hr>
+</div>
+<div id="Azure AD" class="tab-pane fade" markdown="1">
+
+### Azure AD (OIDC)
+
+>**Note**
+>
+> This section is for users who only want to configure Open ID Connect with
+> Azure AD. This connection is a basic OIDC connection, and there are no
+> special customizations available when using it.
+
+1. Sign in to [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} as an administrator and navigate to **Organizations** and select the organization that you want to enable SSO on.
+2. Select **Settings** and select the **Security** tab.
+3. Select an authentication method for **Azure AD**.
+4. In the Identity Provider Set Up, copy the **Redirect URL / Reply URL**.
+
+    ![SSO Azure AD OIDC](/single-sign-on/images/sso-azure-oidc.png){:width="500px"}
+
+5. Sign in to your IdP to complete the IdP server configuration process. Refer to your IdP documentation for detailed instructions.
+
+    > **Note**
+    >
+    > The NameID is your email address and is set as the default.
+    > For example: yourname@mycompany.com.
+
+6. Complete the fields in the **Configuration Settings** section and click **Save**. If you want to change your IdP, you must delete your existing provider and configure SSO with your new IdP.
+
+    ![SSO Azure3](/single-sign-on/images/sso-azure3.png){:width="500px"}
+
+7. Proceed to **add your domain** before you test and enforce SSO.
+
+## Domain control
```

Comment:
Should the `Domain control` section be moved below the tabs? Currently it would only show for Azure.

Reply:
Agree, good catch